### PR TITLE
Make haproxy heath check ip and routing listen ip configurable

### PR DESCRIPTION
**Pull request by aemengo**
Allows the routing release components' network interfaces to be configurable

This is related to the PR here:
https://github.com/cloudfoundry/gorouter/pull/154

### CODE REVIEW

File: jobs/haproxy/spec [hunk 1]
```diff
@@ -21,3 +21,7 @@ properties:
   haproxy.health_check_port:
     description: "Load balancer in front of TCP Routers should be configured to check the health of TCP Router instances by establishing a TCP connection on this port"
     default: 80
+
+  haproxy.health_check_ip:
+    description: "IP for haproxy health check"
+    default: ""
```
Comment (shalako) [inline jobs/haproxy/spec]: should be 0.0.0.0

**Comment by shalako** [comment]
Thank you for resubmitting with the requested changes. Moving to backlog for eng team review.

**Comment by shashwathi** [comment]
Hi @aemengo : 
Thank you for submitting the PR. 

I have couple of questions about the changes to routing-api spec. 
- I understand changing interface for gorouter and TCP router but why is routing api interface change required. Can you explain your use case? 
- Why is `etcd.servers` property moved to CTL script? Routing api consumes `etcd.servers` property from [routing-api.yml](https://github.com/cloudfoundry-incubator/routing-release/blob/develop/jobs/routing-api/templates/routing-api.yml.erb#L26-L35) file instead of command line arguments. 

Regards
Shash

**Comment by shashwathi** [comment]
@aemengo  

Closing this PR due to inactivity. Feel free to create another PR if you still want this feature. 

Regards
Shash

**Pull request closed without merging**